src/app/static/src/tests/components/projects/projectHistory.test.ts

```diff
@@ -77,9 +78,13 @@ describe("Project history component", () => {
         expectTranslated(v.at(4),"Load", "Charger", store);
         expect(v.at(4).find("a").attributes("href")).toBe("");
         expectTranslated(v.at(5), "Delete", "Supprimer", store);
-        expectTranslated(v.at(6),"Copy to a new project", "Copier dans un nouveau projet", store);
-        expect(v.at(6).find("a").attributes("href")).toBe("");
-
+        if (switches.copyProject) {
+            expectTranslated(v.at(6), "Copy to a new project", "Copier dans un nouveau projet", store);
```

Comment:
I like that you can check both translations in one line :)

Reply:
this is our new test util method for testing translations - motivated by your more rigorous approach to testing translations than we had taken in the past!